### PR TITLE
Merge 21.3.0.1 to trunk

### PR DESCRIPTION
**Pull request by oguzkocer**
Includes the `release-toolkit` update to `6.1` in 66eb6bee1d3fa1aac3803cba27c1d0dc2e4be483 as well as changes from https://github.com/wordpress-mobile/WordPress-iOS/pull/19683, https://github.com/wordpress-mobile/WordPress-iOS/pull/19706, https://github.com/wordpress-mobile/WordPress-iOS/pull/19677 & #19693

### CODE REVIEW

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 21.3
 -----
+* [***] Adds a smooth, opt-in transition to the Jetpack app. [#19714]
```
Comment (mokagio) [inline RELEASE-NOTES.txt:7]: Not sure if this has been called out already or not, but wanted to make it explicit that this new release note entry _will not_ be included in the editorialized release notes, which have already been merged via https://github.com/wordpress-mobile/WordPress-iOS/pull/19698.

Part of me feels like this is a missed opportunity, in particular given this is a crucial new step forward in our shift toward Jetpack. But, I looked at the Android release notes and saw they mention the transition [with a one liner](https://github.com/wordpress-mobile/WordPress-Android/blob/0cb9dc18f7ff7bc4f34dbb931af3f4658853517a/WordPress/metadata/release_notes.txt#L1). 

Granted, Android release notes are way more constrained than iOS ones, but that made me think that if we can allow ourselves to ship the Android version without much fanfare on the Play Store, the same goes for the iOS one.

**Review by mokagio — APPROVED** [review]
- [x] Updates to the localization files (`.strings` and Fastlane metadata) automatically pulled for the new translations that have already been approved in GlotPress
- [x] Version update in `.xcconfig`
- [x] Diffs from the PRs that made it into this beta

File: RELEASE-NOTES.txt (hunk 1 above)
Comment (mokagio) [reply RELEASE-NOTES.txt:7]: > this new release note entry _will not_ be included in the editorialized release notes, which have already been merged...

And, to be more precise, the Weekly Kit including those changes has already been sent. Internal reference p4DzFg-1KE-p2.

That was < than two hours ago at the time of writing, so I suppose there _could_ be time to rectify—circumventing the editorialization process by, for example, adopting the same copy [as Android](https://github.com/wordpress-mobile/WordPress-Android/blob/0cb9dc18f7ff7bc4f34dbb931af3f4658853517a/WordPress/metadata/release_notes.txt#L1). As I mentioned above, I don't think that's necessary.

cc @guarani and @hypest for an additional opinion.

Comment (hypest) [reply RELEASE-NOTES.txt:7]: Thanks for the headsup @mokagio!

I'm probably missing context, so it's hard for me to fully understand: Did the `Adds a smooth, opt-in transition to the Jetpack app.` text miss the code freeze, ending up in not being included?

I'd prefer if we include it.

Comment (mokagio) [reply RELEASE-NOTES.txt:7]: > Did the `Adds a smooth, opt-in transition to the Jetpack app.` text miss the code freeze, ending up in not being included?

Correct.

> I'd prefer if we include it.

I'll P2 about this internally.

Comment (mokagio) [reply RELEASE-NOTES.txt:7]: Actually, before doing that, we're better have the new copy merged and ready for upload: https://github.com/wordpress-mobile/WordPress-iOS/pull/19732

**Pull request merged**